frontend/src/data/variables/CovidProvider.ts

```diff
@@ -71,8 +71,9 @@ class CovidProvider extends VariableProvider {
             })
             .resetIndex();
 
-    if (breakdowns.filterFips) {
-      df = df.where((row) => row.state_fips === breakdowns.filterFips);
+    if (breakdowns.filterFips !== undefined) {
+      const fips = breakdowns.filterFips as Fips;
```

Comment:
Does this need the cast? Usually typescript can figure out from the if() check

Reply:
it is actually throwing typescript errors without casting. :/ 
it seems like the first if statement is okay but inside the if/else it's throwing typescript errors :(